gen_cert.py

```diff
@@ -476,7 +500,6 @@ def _generate_certificate(
             (font_string, font_file, None),
             ('Arial Unicode', 'Arial Unicode.ttf', None),
         ]
-        fontlist = self.fontlist
```

Comment:
I'm asked to make unrelated changes quite often.  Unused variables can cause a bunch of issues too.

Your example doesn't make sense in this context.  Why would you grep for self.fontlist when fontlist is scoped to _generate_certificate.  self.fontlist or certificate.fontlist were not changed.